packages/variants/README.md

```diff
@@ -1,11 +1,15 @@
 # [variants](http://dev.office.com/fabric)
 
-Variants are themes based on another theme, subthemes which share the same colors as the original theme, but with those colors used in different places. A variant is still a full theme in and of itself.
+Variants are [themes](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/ghdocs/Theming.md) generated from an existing theme (as opposed to [from raw colors](https://developer.microsoft.com/en-us/fabric#/styles/themegenerator)). A variant will share the same colors as the original theme it was generated from, but will use those colors differently. For example, the background color and text color might be swapped. Variants can be used to highlight or de-emphasize portions of the page.
 
-This project creates variants of a given theme, which are intended for use with the Customizer component in office-ui-fabric-react. These subthemes allow you to highlight or de-emphasize sections of the page with colors that still match the current theme.
+This project provides helper functions to create variants of a given theme, which are intended for use with the [Customizer](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/utilities/src/Customizer.tsx) component. Each function takes in an [IPartialTheme](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/styling/src/interfaces/ITheme.ts) and returns an [ITheme](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/styling/src/interfaces/ITheme.ts).
```

Comment:
Add a little something about how an ITheme will have default values not specified by IPartialTheme.

Reply:
isn't that implied by the interface definitions (and the names of the interfaces themselves)?